Traceback (most recent call last):
  File "/home/chenjiannan/anaconda3/envs/py36/bin/jupyter", line 10, in <module>
    sys.exit(main())
  File "/home/chenjiannan/anaconda3/envs/py36/lib/python3.6/site-packages/jupyter_core/command.py", line 230, in main
    command = _jupyter_abspath(subcommand)
  File "/home/chenjiannan/anaconda3/envs/py36/lib/python3.6/site-packages/jupyter_core/command.py", line 133, in _jupyter_abspath
    'Jupyter command `{}` not found.'.format(jupyter_subcommand)
Exception: Jupyter command `jupyter-notebook` not found.
Traceback (most recent call last):
  File "/home/chenjiannan/anaconda3/envs/py36/bin/jupyter", line 10, in <module>
    sys.exit(main())
  File "/home/chenjiannan/anaconda3/envs/py36/lib/python3.6/site-packages/jupyter_core/command.py", line 230, in main
    command = _jupyter_abspath(subcommand)
  File "/home/chenjiannan/anaconda3/envs/py36/lib/python3.6/site-packages/jupyter_core/command.py", line 133, in _jupyter_abspath
    'Jupyter command `{}` not found.'.format(jupyter_subcommand)
Exception: Jupyter command `jupyter-notebook` not found.
